Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM oven/bun:slim
+
+WORKDIR /app
+
+COPY package*.json ./
+
+RUN bun install
+
+COPY src ./src
+COPY tsconfig.json ./
+
+EXPOSE 4141
+
+ENV NODE_ENV=production
+
+CMD ["bun", "run", "start"]
```

Comment:
Can you turn it into a multi stage build?

You can use this for reference
https://bun.sh/guides/ecosystem/docker